backend/src/main/java/com/bakdata/conquery/models/config/XodusStoreFactory.java

```diff
@@ -213,19 +239,22 @@ private void configureMapper(ConqueryConfig config) {
 		return storages;
 	}
 
-	private List<String> getRelativePathElements(Path path) {
-		ArrayList<String> list = new ArrayList<>();
-		Path relative = getDirectory().relativize(path);
-		for (int i = 0; i < relative.getNameCount(); i++) {
-			list.add(relative.getName(i).toString());
-		}
-		return list;
-	}
-
-	private boolean environmentHasStores(File pathName) {
+	private boolean environmentHasStores(File pathName, Set<String> storesToTest) {
 		Environment env = findEnvironment(pathName);
-		boolean exists = env.computeInTransaction(t -> env.storeExists(StoreInfo.DATASET.getName(), t));
-		env.computeInTransaction(t -> env.getAllStoreNames(t));
+		boolean exists = env.computeInTransaction(t -> {
+			final List<String> allStoreNames = env.getAllStoreNames(t);
+			final boolean complete = allStoreNames.containsAll(storesToTest);
+			if(complete) {
+				log.trace("Storage contained all stores: {}", storesToTest);
+				return true;
+			}
+
+			final HashSet<String> missing = Sets.newHashSet(storesToTest);
```

Comment:
würde ich in log.isWarnEnabled packen, auch weil es dann offensichtlicher ist, dass es nur info ist und nicht logik

Reply:
Gute Idee